### PR TITLE
Bsp/esp box lite

### PR DESCRIPTION
**Pull request by espzav**
# Checklist for new Board Support package or Component

- [x] Component contains License
- [x] Component contains README.md
- [x] Project [README.md](../README.md) updated
- [x] Component contains idf_component.yml file with `url` field defined
- [x] Component was added to CI [upload job](https://github.com/espressif/esp-bsp/blob/master/.github/workflows/upload_component.yml#L17)
- [x] New files were added to CI build job
- [x] CI passing

# Change description
This is my implementation of the ESP-BOX Lite and some changes in examples. The relevant example is `display_audio_photo`, which was extended about the recording and using groups for button navigation.

# Must be done before merge!
- [x] Squash commits
- [x] Increase ESP-BOX version
- [x] Add `bsp_display_get_input_dev` into all BSPs with display
- [x] Add `bsp_audio_codec_speaker_init` and `bsp_audio_codec_microphone_init` into all audio BSPs (At least into ESP-BOX)

### CODE REVIEW

**Comment by espzav** [comment]
Hi @igrr and @tore-espressif, here is my implementation of the ESP-BOX-Lite. 
There is one BIG question:
This cannot be finished before the `esp_codec_dev` wouldn't be as component in IDF Component Registry. How long, we can wait for it? Or could we copy `esp_codec_dev` into BSP???

#How test it?
- Use `display_audio_photo` example
- Change BSP component in YML for `esp-box-lite`
- Copy `esp_codec_dev` folder to `display_audio_photo/components`
- Add `esp_codec_dev` component to `CMakeLists.txt` in `esp-box-lite` component (This step must be solved - add `esp_codec_dev` into IDF Component Registry)
- In demo - there can be navigate via buttons (left/up, ok, right/down). On top tabs, there one click select tabs, then move by left/right and click for select tab. Long click deselect tabs. Same for sliders - one click for select, long click for deselect.

**Comment by espzav** [comment]
The esp_codec_dev is done and now this is ready for review. @tore-espressif PTAL

**Review by tore-espressif — COMMENTED** [review]
Nice work @espzav 

One more comment, probably not related to this PR: When I was building the `display_audio_photo` example, the `esp_lcd_touch` was added to the project (as dependency of esp_lvgl_port). We can fix it in another PR

**Comment by espzav** [comment]
@tore-espressif Thank you for the review! I fixed and updated all things, that you mention in comments. 
Created new tasks: BSP-289, BSP-290

**Comment by espzav** [comment]
Updates:
- Added to bsp_ext.py
- Added `bsp_display_get_input_dev` into all BSPs

File: esp-box/esp-box.c [hunk 1]
```diff
@@ -29,10 +30,20 @@ _Static_assert(CONFIG_ESP_LCD_TOUCH_MAX_BUTTONS > 0, "Touch buttons must be supp
 /** @endcond */
 
 static lv_disp_t *disp;
+static lv_indev_t *disp_indev = NULL;
 static esp_lcd_touch_handle_t tp;   // LCD touch handle
+static const audio_codec_data_if_t *i2s_data_if = NULL;  /* Codec data interface */
+static i2s_chan_handle_t i2s_tx_chan = NULL;
+static i2s_chan_handle_t i2s_rx_chan = NULL;
+static bool i2c_initialized = false;
```
Comment (tore-espressif) [inline esp-box/esp-box.c:38]: Nitpick: Could we make this variable static insied the `bsp_i2c_init` function?

**Review by tore-espressif — APPROVED** [review]
Only one nitpick, otherwise LGTM!

**Comment by espzav** [comment]
@tore-espressif Thank you for the review!

**Pull request merged**